docs/Reference/TypeScript.md

```diff
@@ -687,6 +687,144 @@ Or even explicit config on tsconfig
 }
 ```
 
+#### Decorators
+
+Fastify's decorator system allows you to extend the Fastify instance, request, and
```

Comment:
I know the original docs had it, but could you please remove the [second-person "you" usage](https://fastify.dev/docs/latest/Guides/Style-Guide/#when-to-avoid-the-second-person-you-as-the-pronoun) throughout this?